### ISSUE TITLE
Issue downloading model

### ISSUE THREAD

**Issue by adithya-shetty** [MattKleinsmith/void-detector#3]
I tried to download the model using the following command:
`python3 download_model.py`
But I'm getting  the following error regarding permission of google drive:
`Permission denied: https://drive.google.com/uc?id=10CUc12hH3Us9jQGm5I4x8BiszXpBbP7V
Maybe you need to change permission over 'Anyone with the link'?`
Could you please fix this?
Also could you provide the command to build the docker image and run the container. 
Thanks

**Comment by sujit-dn** [comment]
Yup, i am also facing issues while downloading model. Its not available. Pls fix link so that we can experiment with model.
